ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -111,6 +112,26 @@ static StatsSnapshot convertQuotaToStatsSnapshot(Map<String, Map<String, Long>>
     return new StatsSnapshot(totalSize, accountValidSizeMap);
   }
 
+  /**
+   * Convert a given nested {@link Map} of accountId to containerId to valid size to its corresponding
+   * container {@link StatsSnapshot} object. The container snapshot does not have account level, each container is
+   * uniquely identified by "Account[]_Container[]" pair.
+   * @param utilizationMap the nested {@link Map} to be converted
+   * @return the corresponding {@link StatsSnapshot} object.
+   */
+  static StatsSnapshot convertStoreUsageToContainerStatsSnapshot(Map<String, Map<String, Long>> utilizationMap) {
+    Map<String, StatsSnapshot> containerValidSizeMap = new HashMap<>();
+    long totalSize = 0;
+    for (Map.Entry<String, Map<String, Long>> accountEntry : utilizationMap.entrySet()) {
+      for (Map.Entry<String, Long> containerEntry : accountEntry.getValue().entrySet()) {
+        totalSize += containerEntry.getValue();
+        containerValidSizeMap.put(accountEntry.getKey() + "_" + containerEntry.getKey(),
```

Comment:
i think you should declare the separator ("_") as a variable in a common place. Also I think you should use "__" or "___" (two or three underscores) as the separator in case the account name itslef has underscores

Reply:
I will use three underscores as separator. Thanks for suggestion.